_examples/ceph-source-example.adoc

```diff
@@ -31,21 +31,34 @@ https://github.com/ceph/ceph-ansible[Ceph Ansible playbooks]
 In case you need to configure your own Ceph environment, follow the
 steps below:
 
-1. Clone https://github.com/ceph/ceph-ansible[ceph-ansible] GitHub repo.
-2. Rename `vagrant_variables.yml.sample` to `vagrant_variables.yml`
-   and edit the file by changing to the following snippet:
+1. Download `ceph-nano` in your computer:
 
-   ...
-   # DEFINE THE NUMBER OF VMS TO RUN
-   mon_vms: 1
-   osd_vms: 3
-   mds_vms: 0
-   rgw_vms: 1
-   ...
+  ...
+  curl -L https://github.com/ceph/cn/releases/download/v1.6.1/cn-v1.6.1-b900468-linux-amd64 -o cn && chmod +x cn && mv cn /usr/local/bin
```

Comment:
i think we should change this command,

first the formatting should be a little tighter to make it easier to read on the screen, example

```
curl -L https://github.com/ceph/cn/releases/download/v1.6.1/cn-v1.6.1-b900468-linux-amd64 -o cn \
&& chmod +x cn
```

second, i don't think we should advise people to move the binary to `/usr/local/bin` as that will require a superuser privilege. we should just tell them to execute locally (`./cn`)